src/ui/ColorPicker.scss

```diff
@@ -0,0 +1,113 @@
+.widget-color-picker {
+    .sketch-picker, .compact-picker {
+        input {
+            width: 100% !important;
+        }
+    }
+
+    .material-picker {
+        height: auto !important;
+    }
+
+    .block-picker {
+        box-shadow: rgba(0, 0, 0, 0.3) 0px 0px 2px, rgba(0, 0, 0, 0.3) 0px 4px 8px !important;
+    }
+
+    &.widget-color-picker-disabled {
+        position: relative;
+    }
+
+    .widget-color-picker-overlay {
+        z-index: 50;
+        background: rgba(255,255,255,0.5);
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+    }
+
+    .circle-picker > span {
+        margin: 8px;
+    }
+
+    .circle-picker > span > div {
+        margin: 0 !important;
+    }
+
+    .circle-picker {
+        box-shadow: rgba(0, 0, 0, 0.25) 0px 1px 4px;
+        background: rgb(255, 255, 255);
+        padding: 14px 0 14px 14px;
+        margin-bottom: 0px !important;
```

Comment:
Prefer if possible, it without `!important`? Can we use more specific selectors?

Reply:
The component already has an inline styling